.github/scripts/template-zip-autogen-v3.sh

```diff
@@ -14,15 +14,18 @@ TEMPLATE_OUTPUT_DIR=$1
 mkdir -p ${TEMPLATE_OUTPUT_DIR}
 
 TEMPLATE_LIST=$(jq -r '.templatesV3[]' ./templates/package.json)
```

Comment:
Seems TEMPLATE_LIST is not used in following script and it can be deleted from package.json?

Reply:
Sure.